packages/bpk-component-datepicker/src/BpkDatepicker.js

```diff
@@ -40,6 +40,17 @@ class BpkDatepicker extends Component {
     };
   }
 
+  componentDidUpdate(prevProps) {
```

Comment:
I think `getDerivedStateFromProps` would be a better alternative - see [this React blog](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html). It would also mean you don't have to disable the `react/no-did-update-set-state` rule.

Reply:
That would mean bumping the peerDep for react though to 16.4 (currently [16.0](https://github.com/Skyscanner/backpack/blob/fbe1d4ddb7643025ba366a5ad03e924629e4117c/packages/bpk-component-datepicker/package.json#L13)).